tests/memory/enc/stress.c

```diff
@@ -172,7 +172,7 @@ static void _run_malloc_test(size_t size)
 
 void init_malloc_stress_test()
 {
-    srand(time(NULL));
+    srand((int)time(NULL));
```

Comment:
Interesting, apparently the time function returns a 'time_t' type, which is up to an implementation to specify its format. I think most implementations use a typedef for time_t to be a "long int", which is probably why the compiler threw a warning here since we're truncating the value. Seems fine to me to do this in a test like this!